src/classes/mech/Mech.ts

```diff
@@ -252,10 +252,10 @@ class Mech implements IActor {
   }
 
   public get Size(): number {
-    let size = Bonus.Int(this._frame.Size, 'size', this)
+    let size = this._frame.Size >= Rules.MaxFrameSize ? this._frame.Size : Bonus.Int(this._frame.Size, 'size', this)
```

Comment:
This looks like we are now ignoring the check against Rules.MaxFrameSize. If that is the case, why check against it still?

Reply:
This check is basically coding for Fomorian Frame.  Fomorian Frame has no effect if the Frame is size 3 or greater, but will apply the size bonus if the frame is less than size 3.  Don't know if CompCon can handle cases greater than Size 4, but that'll fall on the shoulders of Homebrew creators instead of us.